charts/sdf-generation/values.yaml

```diff
@@ -0,0 +1,84 @@
+replicaCount: 1
+
+image:
+  repository: ghcr.io/thefittingroom/sdf-generation-worker
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: 0.0.1
+
+imagePullSecrets:
+  - name: docker-registry
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations:
+    eks.amazonaws.com/role-arn: 'arn:aws:iam::602373492013:role/sdf-generation-dev'
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: sdf-generation
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 8000
+
+ingress:
+  enabled: false
+  className: ""
+  annotations: {}
+  hosts:
+    - host: chart-example.local
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+
+#resources:
+#  limits:
+#    nvidia.com/gpu: 1
+
+env:
+  LOGER_LEVEL: INFO
```

Comment:
How does your app know which S3 bucket to use if its not supplied here? I would make your app use the env variable to know which s3 bucket to operate on so it can be injected here.